spring-batch-docs/asciidoc/scalability.adoc

```diff
@@ -238,19 +241,19 @@ The `Step` on the right in this case is the "remote" Slave, so
 </step>
 ----
 
-Similar to the multi-threaded step's throttle-limit
-    attribute, the grid-size attribute prevents the task executor from
+Similar to the multi-threaded step's `throttle-limit`
+    attribute, the `grid-size` attribute prevents the task executor from
     being saturated with requests from a single step.
 
-There is a simple example which can be copied and extended in the
-    unit test suite for Spring Batch Samples (see
+There is a simple example that can be copied and extended in the
+    unit test suite for https://github.com/spring-projects/spring-batch/tree/master/spring-batch-samples[Spring Batch Samples] (see
```

Comment:
Got a little lost here.   Could we use this link to get them closer: https://github.com/spring-projects/spring-batch/tree/master/spring-batch-samples/src/main/resources/jobs

Also should we replace `*PartitionJob.xml` with `Partition*Job.xml`

I can change this on merge if you agree.

Reply:
Yes, please.